src/translations/en_FR.json

```diff
@@ -61,7 +61,6 @@
   "SIDEBAR_ADD_DISCOUNT_CELL_PLACEHOLDER": "Your code",
   "SIDEBAR_STARTDATE_CELL_VALUE_NEW": "Today",
   "SIDEBAR_STARTDATE_CELL_VALUE_SWITCHER": "When your old one expires",
-  "CHECKOUT_EMAIL_LABEL": "Email",
```

Comment:
Why is this key deleted!? @mehdielket

Reply:
I've updated translations so we have this key back.